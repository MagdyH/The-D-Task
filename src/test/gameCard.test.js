import React from 'react';
import renderer from 'react-test-renderer';
import GameCard from '../components/gameCard';


describe('GameCard component', () => {
    it('should render snapshot', () => {
      const component = renderer.create(<GameCard />);
  
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

});