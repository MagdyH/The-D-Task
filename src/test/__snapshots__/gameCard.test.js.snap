// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameCard component should render snapshot 1`] = `
<div
  className="my-4"
  style={
    Object {
      "backgroundColor": "#EEF0F7",
      "color": "#7F7F7F",
    }
  }
/>
`;
